activerecord/lib/active_record/connection_adapters/postgresql/schema_creation.rb

```diff
@@ -24,6 +27,12 @@ def visit_CheckConstraintDefinition(o)
             super.dup.tap { |sql| sql << " NOT VALID" unless o.validate? }
           end
 
+          def visit_CreateIndexDefinition(o)
+            super.dup.tap do |sql|
+              sql << "INCLUDE (#{quoted_include_columns(o.index.include)})" if supports_index_include? && o.index.include
```

Comment:
I think, this should be better done in the `abstract/schema_creation.rb` base class since SQL Server also [supports](https://www.sqlservertutorial.net/sql-server-indexes/sql-server-indexes-with-included-columns/) include indexes with the same syntax.

Reply:
I should have commented this in the PR description. I saw SQL Server supports INCLUDE, but I don't have any experience with SQL Server, so my intent was to implement an initial iteration on Postgres and leave things open for SQL Server in a followup. Not sure how the maintainers like to handle things like this, but I'm a big fan of small iterations for features.

Reply:
Ah, I see another comment from Eileen regarding this -https://github.com/rails/rails/pull/44803#discussion_r839473577.
Yes, I think this is too postgres specific, better to keep it where it is.